keps/369-job-interface/README.md

```diff
@@ -0,0 +1,329 @@
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+From day 0 in Kueue, we natively support Kubernetes Job by leveraging the capacity of [_suspend_](https://kubernetes.io/docs/concepts/workloads/controllers/job/#suspending-a-job),
+this helps us to build a multi-tenant job queueing system in Kubernetes, this is attractive to other job-like applications
+like MPIJob, who lacks the capacity of queueing.
+
+The good news is Kueue is extensible and simple to integrate with through the intermediate medium, we named _Workload_ in Kueue,
+what we need to do is to build a controller to reconcile the workload and the job itself.
+
+But the complexity lies in developers who are familiar with job-like applications may have little knowledge of the
+implementation details of Kueue and they have no idea where to start to build the controller. In this case, if we can provide an
+interface which defines the default behaviors of the Job, and serve the Kubernetes Job as a standard template, it will do them
+a great favor.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- Define an interface which shapes the default behaviors of Job
+- Make Kubernetes Job an implementation template of the interface
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+- Integrate with other job-like applications
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+We collected feedbacks from the community about how to fully integrate Kueue with MPIJob,
+see [#499](https://github.com/kubernetes-sigs/kueue/issues/499).
+
+### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+- Job interface here is more like a template and tell developers what to do, but even without
```

Comment:
To be honest I'm lost here. Is it a template, a hint, a set of best practices or a library?